recipe/build.sh

```diff
@@ -2,8 +2,18 @@
 set -e
 set -x
 
+if [[ ! -z "$mpi" && "$mpi" != "nompi" ]]; then
+  export CONFIGURE_ARGS="--with-mpi=${PREFIX} ${CONFIGURE_ARGS}"
+  export CC=mpicc
+  export CXX=mpic++
+  export FC=mpif90
+  export F77=mpif77
+  export F90=mpif90
```

Comment:
New best guess at compiler variables based on:
https://bitbucket.org/fathomteam/moab/src/1d583ea0419a7658c9c00bd0e1b421391918d50f/bitbucket-pipelines.yml#lines-9

I had used `mpifort` based on the hdf5 recipe:
https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/build.sh#L10

Reply:
Yay!  That seems to have done it!

Reply:
You could also specify the compiler environment as part of the `CONFIGURE_ARGS` if you want to make things concise. For example 

`export CONFIGURE_ARGS="CC=mpicc CXX=mpic++ F77=mpif77 FC=mpif90 --with-mpi=${PREFIX} ${CONFIGURE_ARGS}"`